bin/jira

```diff
@@ -3,10 +3,12 @@
 require 'jira'
 begin
   Jira::CLI.start
+rescue Faraday::Error => e
+  puts "JIRA failed to REST, you may need to 'jira install'"
```

Comment:
This error message don't make much sense, but I can change it in a separate commit.